### PR TITLE
`um` as a real unit

### PR DESCRIPTION
**Pull request by mondoreale**
In this PR I make `um` a unit that you can use in your CSS. Unlink current `rem`, it always represents 16px.

Adding all of you as reviewers cause it'd be nice for all of us to be aware of its existence.

---

_`um` stands for Unit of measurement._

### CODE REVIEW

File: app/src/userpages/components/Header/Tab/tab.pcss [hunk 1]
```diff
@@ -1,11 +1,11 @@
 .root {
   font-family: var(--mono);
-  font-size: calc(0.75 * var(--um));
+  font-size: 0.75um;
```
Comment (mondoreale) [inline app/src/userpages/components/Header/Tab/tab.pcss]: It's simply easier to use than the actual `calc`+`var`.

Comment (timoxley) [reply app/src/userpages/components/Header/Tab/tab.pcss]: cool

File: app/scripts/postcss/ums.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const postcss = require('postcss')
+
+// `um` is a unit representing 16px (`--um` variable). It's temporary. Current `rem` is
+// contextual (depends on screen size). Once we fix that (make `rem` constant) we can
+// replace all ums with rems. Simple as that.
```
Comment (timoxley) [inline app/scripts/postcss/ums.js]: > Once we fix that (make `rem` constant)

What's preventing us from doing this? Bootstrap?

Comment (mondoreale) [reply app/scripts/postcss/ums.js]: Altho bootstrap may have something to do with that (needs checking), I'm more concerned about the second thing you mentioned – things that will break if we simply force a single value for `rem`.

**Comment by juhah** [comment]
A bit late to the party but this seems like a nice improvement 👍

**Pull request merged**